include/lmms_math.h

```diff
@@ -34,7 +34,7 @@
 #include <cmath>
 using namespace std;
 
-#if defined (LMMS_BUILD_WIN32) || defined (LMMS_BUILD_APPLE) || defined(LMMS_BUILD_HAIKU)  || defined (__FreeBSD__)
+#if defined (LMMS_BUILD_WIN32) || defined (LMMS_BUILD_APPLE) || defined(LMMS_BUILD_HAIKU)  || defined (__FreeBSD__) || defined(__OpenBSD__)
```

Comment:
Probably time for us to roll `IF(NOT CMAKE_COMPILER_IS_GNUCC) ...` here, as this list is getting quite long and AFAIK, these are all GNU compat helpers.